gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
To be more specific: the dependency hell you are experiencing is solved w/ the function composition of the new task system. Doing a dep graph was a bad idea.